modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -22,83 +22,240 @@ Runtime Fabric enables you to specify the level of severity of the message writt
 | priority:SYSTEM
 
 | CONSOLE
-| List message about console events such as setting the objectstore
+| List messages about console events such as setting the object store.
 | priority:CONSOLE
 
 | WARN
-| List warning messages
+| List warning messages.
 | priority:WARN
 
 | DEBUG
-| List debugging messages
+| List debugging messages.
 | priority:DEBUG
 |===
 
 Log levels are specified per Mule application or API proxy during deployment.
 
+[WARNING]
+Because log levels are specified when deploying a Mule application or an API proxy, you cannot change the log levels after deployment.
 
+== View Logs from an Application
 
+Ops Center shows a stream of logs generated by applications and services running on Runtime Fabric. This is useful when log forwarding is not configured.  
 
-[WARNING]
-Log levels are specified when deploying a Mule application or an API proxy. After deployment you cannot change the log levels.
+. From Ops Center, navigate to *Logging*.
+. Select *Kubernetes*.
+. Select the *Pods* tab.
+. In the drop-down list adjacent to the search input area, select the ID of the environment where the application is deployed.
+. Locate the pod name that starts with the name of your application.
+. Select that pod name and then select *Logs*.
 
+The *Logs* tab is displayed with a filter applied to your application.
 
-== Viewing Logs
+[NOTE]
+To view the latest logs, select *Refresh*.
 
-Ops Center shows a stream of logs outputted by applications and services running on Runtime Fabric. Navigate to "Logging" on the left of the page to view the logging interface.
+=== Filters
+
+There are two levels of filters for drilling down in the logs:
+
+* The `_Containers_` filter filters by container name.
+* The `_Pods_` filter filters by pod name.
++
+This is useful for specifying application names that are followed by a wildcard (`%`).
 
-=== View Logs from an Application
+== Forward Logs to External Services
 
-With Ops Center, you can view the logs from a deployed application. This can be useful in cases where log forwarding is not set up.
+Runtime Fabric supports log forwarding to:
 
-. On Ops Center, click on Kubernetes on the left sidebar.
-. Click on the Pods tab.
-. Select the environment ID where the application was deployed on the right dropdown, near the search input.
-. Find the Pod name which begins with the name of your application.
-. Click on the Pod name and select "Logs".
+* Anypoint Monitoring
+* Third-party logging solutions using Fluent Bit log forwarding output plugins:
 
-The page should redirect to the Logs tab with a filter applied to your application.
+** Azure
+** Elasticsearch
+** Graylog Extended Log Format (GELF)
+** Splunk
 
 [NOTE]
-To view the latest logs, click the "Refresh" button on the upper right portion of the page.
+Only one Fluent Bit log forwarding output plugin is supported at any one time. However, you can forward logs to Anypoint Monitoring and one Fluent Bit log forwarding output plugin at the same time.
```

Comment:
Suggestion: "Only *one* external log forwarding configuration is supported at any one time. However, you can forward logs to Anypoint log forwarding and external log forwarding service at the same time."

Reply:
Updated this.